geniza/corpus/models.py

```diff
@@ -894,7 +894,8 @@ def index_data(self):
         index_data.update(
             {
                 "pgpid_i": self.id,
-                "type_s": str(self.doctype) if self.doctype else _("Unknown type"),
+                # type gets matched back to DocumentType object in get_result_document, for i18n
+                "type_s": str(self.doctype) if self.doctype else "Unknown type",
```

Comment:
good catch; should we note explicitly we want to always to index these in english?